third-party/realdds/py/pyrealdds.cpp

```diff
@@ -880,10 +882,15 @@ PYBIND11_MODULE(NAME, m) {
         .def( "server_guid", &dds_device::server_guid )
         .def( "guid", &dds_device::guid )
         .def( "is_ready", &dds_device::is_ready )
+        .def( "is_online", &dds_device::is_online )
```

Comment:
What about `is_offline` and `wait_until_offline`?

Reply:
OK ,will add